cpp/src/column/column_view.cpp

```diff
@@ -20,6 +20,8 @@
 #include <cudf/utilities/error.hpp>
 #include <cudf/utilities/traits.hpp>
 
+#include <cudf_test/column_wrapper.hpp>
```

Comment:
I don't think cudf should depend on cudf_test. It should be the other way around (and is). If this is just to get `make_counting_transform_iterator`, then just move that into a cudf header.